R/compat-types-check.R

```diff
@@ -127,13 +131,17 @@ check_number_decimal <- function(x,
 
 check_number_whole <- function(x,
                                ...,
+                               min = -Inf,
+                               max = Inf,
```

Comment:
Could you pass `allow_infinite` here too? I know that technically Inf is not an integer, but it's often use to have an interface that usually requires integers, but uses `Inf` to mean an "unbounded number".

Reply:
For example, I think `n` in `slice_max()` is like that - which admittedly is probably a little weird, because why are you even slicing at that point